draft-ietf-gnap-core-protocol.md

```diff
@@ -3374,7 +3378,7 @@ Content-Digest: sha-256=...
 }
 ~~~
 
-An attempt to change the `proof` method or parameters, including an attempt to rotate the key of a bearer token (which has no key), MUST result in an error returned from the AS.
+An attempt to change the `proof` method or parameters, including an attempt to rotate the key of a bearer token (which has no key), MUST result in a "request_denied" error code returned from the AS.
```

Comment:
I hesitated here, tbh. That's a sort of request_denied (after all all POST are requests), but there could also be a specific type because it doesn't apply as part of a GNAP request per say.

Anyway, it was more a first test to actually have some systematic pass over all error codes.

Reply:
I propose "invalid_rotation"